app/topics/topic-management/manage-attachments/manage-attachments.component.html

```diff
@@ -15,43 +15,41 @@ <h3 md-line>{{ attachment.name }}</h3>
         {{ 'by user' | translate }} {{ attachment.user }}
         <span *ngIf="attachment.license != ''">- {{ attachment.license }}</span>
       </p>
-      <div class="buttons">
+      <md-card-actions align="end">
```

Comment:
Is the class `.buttons` used somewhere else? if not it can be deleted from the css file